src/test/components/ProfileThreadTracingMarkerOverview.test.js

```diff
@@ -12,68 +12,69 @@ import { storeWithProfile } from '../fixtures/stores';
 import { getProfileWithMarkers } from '../fixtures/profiles/make-profile';
 import ReactDOM from 'react-dom';
 import { getBoundingBox } from '../fixtures/utils';
+import mockRaf from '../fixtures/mocks/request-animation-frame';
 
-jest.useFakeTimers();
-ReactDOM.findDOMNode = jest.fn(() => {
-  // findDOMNode uses nominal typing instead of structural (null | Element | Text), so
-  // opt out of the type checker for this mock by returning `any`.
-  const mockEl = ({
-    getBoundingClientRect: () => getBoundingBox(200, 300),
-  }: any);
-  return mockEl;
-});
+describe('ProfileThreadTracingMarkerOverview', function() {
+  beforeEach(() => {
```

Comment:
I opted to wrap the mock in a `beforeEach`, and put the `it` block into a `describe` block. There is a lot of indentation noise in the diff.